src/containers/Tokens/index.js

```diff
@@ -25,6 +27,17 @@ const Tokens = props => {
       .get(TOP_TOKENS_URL)
       .then(res => {
         const { tokens } = res.data;
+
+        if (res.data.result === 'error') {
```

Comment:
I was thinking that we could keep this as !== success, for all random cases? @natenichols @mvadari

Reply:
`res?.data?.result !== 'success'` should handle cases where either res or data are `undefined`

Reply:
Doesn't really matter - the response is either `success` or `error` for this endpoint

Reply:
Up to you :)

Reply:
> `res?.data?.result !== 'success'` should handle cases where either res or data are `undefined`

Yes, that's better